docs/src/pages/index.js

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
some double license headers here, is this file apache or cc-by-sa?

Reply:
See my other command. I think it's likely both.